rosa_release_notes/rosa-release-notes.adoc

```diff
@@ -0,0 +1,104 @@
+
+ROSA clusters are available at https://console.redhat.com/openshift. With the {cluster-manager-first} application for ROSA, you can deploy OpenShift clusters to either on-premise or cloud environments.
+
+[id="about-this-release_{context}"]
+== ROSA layered and dependent component support and compatibility
+The scope of support for layered and dependent components of ROSA changes independently of the ROSA version. To determine the current support status and compatibility for an add-on, refer to its release notes. For more information, see the xref:../rosa_architecture/rosa_policy_service_definition/rosa-life-cycle.adoc#rosa-life-cycle[{product-title} update life cycle].
+
+[id="rosa-new-features"]
+== New features and enhancements
+
+[id="rosa-product-enhancements"]
+=== Product enhancements
+
+The following list describes the key features of ROSA:
+
+* **ROSA UI wizard**: The new ROSA provisioning wizard UI is now available as an AWS STS-only experience. You can access the new UI wizard at https://console.redhat.com/openshift/create/rosa/wizard.
+
+* **Latest Command Line Interface and New Operator**: ROSA Command Line Interface (CLI) 1.1.11 released on March 9, 2022 and later enabled the creation of ROSA 4.10 clusters. Please ensure you are always using the latest ROSA CLI for the best experience. OpenShift 4.10 has a new operator and cloud network configuration controller. Therefore, a new operator role and policy will be added when you upgrade to OpenShift 4.10, or it will be a part of new cluster deployments. During an upgrade to OpenShift 4.10, you will be prompted by the ROSA CLI to add a new role and policy.
+
+* **Upgrade handling**: The commands `rosa upgrade cluster` and `rosa upgrade account` roles are seamless and backward compatible with all the older ROSA versions and older ROSA clusters. When you run these commands, the old ROSA clusters and their relevant STS roles and policies are updated to the latest versions.
+
+* **New AWS roles supporting the ROSA UI**: The tools to manage the `ocm-role` and `user-role` and use the ROSA provisioning UI are now available in the ROSA CLI as of version 1.2.0. The `ocm-role` and `user-role` are also instrumental in facilitating ROSA STS upgrades done from the {cluster-manager-first} UI only.
+
+[id="rosa-documentation-enhancements"]
+=== Documentation enhancements
+
+The following list describes the key documentation features of ROSA:
+
+* **Getting started with ROSA**: ROSA 4.10 now includes a getting started guide. Getting Started with ROSA defines basic terminology and provides role-based next steps for developers and administrators. The tutorials walk new users through the web console and the OpenShift CLI interfaces. New users can accomplish the following tasks through the Getting Started:
```

Comment:
I think if Getting Started with ROSA is the section title it should be in bold, same for any references to section titles that aren't links

Reply:
"New users can accomplish the following tasks through the Getting Started:"

I think this should be "New users can accomplish the following tasks by using the getting started guide:"

Reply:
The changes are incorporated.